aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -81,9 +106,14 @@ public class GridPlacer : MonoBehaviour
         private GameObject _teleporterPrefab;
 
         /// <summary>
-        /// The random number generator.
+        /// The distance from the floor to the player localpos.
         /// </summary>
-        private Random _random = new();
+        private readonly Vector3 _playerHeightOffset = Vector3.up * 0.7f;
```

Comment:
This will probably break if the player height is changed right? Maybe get the height from the player